packages/pagination/src/PaginationContent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Util } from 'reactstrap';
+import BlockUI from 'react-block-ui';
```

Comment:
may want to include the CSS for BlockUI (though, I'm not 100% since this will be a library and it would require the end-developer's build tool to be able to pick up CSS imports from node_modules)